azure-kusto-ingest/tests/e2e.py

```diff
@@ -126,8 +124,8 @@ def dm_kcsb_from_env() -> KustoConnectionStringBuilder:
     return KustoConnectionStringBuilder.with_aad_application_key_authentication(dm_cs, app_id, app_key, auth_id)
 
 
-def clean_previous_tests(engine_client, database, table):
-    engine_client.execute(database, ".drop table {0} ifexists".format(table))
+def teardown_module():
```

Comment:
Is there some pytest convention that makes this API be called after the test is run?

Reply:
Exactly: https://docs.pytest.org/en/stable/xunit_setup.html